### ISSUE TITLE
Invite to the slack channel

### ISSUE THREAD

**Issue by askar1680** [facebook/buck#2669]
Hi

Can you please add @yandex-team.ru to the email whitelist.

Thanks

**Comment by rajyengi** [comment]
Added - use this link to sign up: https://join.slack.com/t/buckbuild/signup.

**Comment by rajyengi** [comment]
Added the email domain

**Comment by JoshuaN129** [comment]
There aren’t enough words to describe or explain my wonderful experience with CYBERCREDITGURU a US based Russian h a c k e r s. In 2018, I had 205 BTC in my blockchain wallet but out of my carelessness I misplaced my password and recovery phrase. I really don’t know how to go about it until early this year, I came across a 5star review about them in a forum online by one Mrs. Kimberly and I reached out to them via text: +1 (650) 439 0624 and CYBERCREDITGURU (AT) GMAIL DOT COM. They responded immediately and helped recovered my lost BTC wallet within 4 days. Thanks to Team CyberGuru. I also had an extremely poor credit report of 403 with derogatory; late payment, repossession, collections and foreclosure. They also cleared all negative items from my credit report and upgraded my FICO score to 810 across all three major bureaus. Now with no doubts I can say 2022 is a great year for me as I am a proud owner of a new house and a car. All thanks to CYBERCREDITGURU once again. They are my messiah! I confidently recommend them to you.
